src/Factory.php

```diff
@@ -24,13 +24,6 @@ class Factory
     /**
      * Constructor.
      *
-     * @param array $config
-     *   An array of configurations. You need at least the 'key'.
```

Comment:
This seems like it could be valuable?

Reply:
Added comment below. We could also check and throw an exception if it isn't there. What do you think?